core/domain/collection_domain.py

```diff
@@ -780,23 +786,12 @@ def get_next_exploration_id_in_sequence(self, current_exploration_id):
         return exploration_just_unlocked
 
     def get_nodes_in_playable_order(self):
```

Comment:
Do we even need this function anymore? Should we just remove it and replace it with self.nodes at all its callsites?